docs/user-guide/join-a-jitsi-meeting.md

```diff
@@ -4,11 +4,17 @@ title: Join a Jitsi Meeting
 sidebar_label: Join a Jitsi Meeting
 ---
 
+# Join by using a Jitsi link
 
-1. When visiting a Jitsi Meeting link, your browser may first ask you to grant microphone and/or camera access.
-   Please refert to the browser's documentation for details (e.g.
+People invite each other to Jitsi meetings mostly by sending a link.
+
+1. If you have received such an invite link from a trusted source,
+   copy it into your browser and press <kbd>Enter</kbd>/<kbd>Return</kbd>.
+2. Your browser may first ask you to grant microphone and/or camera access.
+   If you trust the person who invited you, confirm this access request.
```

Comment:
This is a bit tricky :thinking: ...but I would add some more information like:
1. Microphone and Camera are not strictly necessary (`Only allow, if you want others to see and hear you`) and can be used seperately (e.g. microphone only).
2. Trust in server 
3. Trust in other people in the room

Also maybe use the word `allow` (instead of `confirm`), as these are the terms used at least in firefox (if i remember correctly).
I think it might be easier for people to understand.